internal/ethapi/transaction_args.go.orig

```diff
@@ -0,0 +1,422 @@
+// Copyright 2021 The go-ethereum Authors
```

Comment:
I don't think we want these `.org` files.

Reply:
ah damn